source/manual/republishing-content.html.md

```diff
@@ -15,15 +15,19 @@ This process varies per app and requires
 
 ## Whitehall
 
-If the document is in Whitehall, there is a Rake task you can run:
+If the documents are in Whitehall, there are Rake tasks you can run as outlined below. Try to pick the one most focused to the scope of what you need to republish to avoid unnecesary load. You can monitor the effect on the publishing queue via these dashboards:
```

Comment:
```suggestion
If the documents are in Whitehall, there are Rake tasks you can run as outlined below. Try to pick the one most focused to the scope of what you need to republish to avoid unnecessary load. You can monitor the effect on the publishing queue via these Grafana dashboards:
```